examples/introduction.json

```diff
@@ -0,0 +1,3 @@
+{"Number" : 1,
+ "String" : "two",
+ "List": ["containing", "different", "types", 35.4]}
```

Comment:
What is this file for? It is not used in the code as far as I see.

Reply:
It's to give an example input file for the "sum" and "reverse" tests, which take an input file on the command line. I have added a note to that effect in the source.